docs/development/building/Building-in-Windows.md

```diff
@@ -25,7 +25,11 @@ wsl.exe --update
 shutdown.exe /r /f /t 5
 ```
 
-After reboot, open an elevated terminal again and run the following commands:
+After reboot, continue with the following steps.
+
+By default WSL is configured for 2GB memory usage. Use the Start-menu to search for and open `WSL Settings`. Increase WSL's `Memory Size` to at least `4096`. (Note, some targets will compile with only 2048, but newer G4 and H7 may not.)
```

Comment:
What is the size of the VM you are testing this in ?
See also https://learn.microsoft.com/en-us/windows/wsl/wsl-config#main-wsl-settings

Reply:
4GB vRAM with 4GB WSL.  H7 and G4 worked, now testing `make all` (slow).

Reply:
maybe i'll convert this to a `:::tip` or `:::note`.

Reply:
or `:::warning` :)